### ISSUE TITLE
Make SNVs and Indels requests asynchronous to avoid page expiring

### ISSUE THREAD

**Issue by northwestwitch** [Clinical-Genomics/scout#4010]
It is allowed to look for variants with rank score >=5, but if you go to low with this score it's quite likely that the page times out

**Comment by dnil** [comment]
Yes... Though that depends on what you set the sparse index to, which could in principle also be changed outside the config. We have different bounds for stage and prod e.g. 😊 If you want to limit it, ideally check with the db for what that sparse index bound is - or perhaps good enough, introduce a constant that is used both in the db index setup and the interface! 😄
